spec/rails_helper.rb

```diff
@@ -133,22 +133,6 @@
 
   SpecSupport.hooks(config, chrome_driver)
 
-  config.filter_run_excluding(twilio_verify_specs: true) unless RadConfig.twilio_verify_enabled?
```

Comment:
I can't remember removing these, did you do this @garyfoster ?

Reply:
I removed them all as they are not needed, they're only needed in the shared version of rails_helper that gets copied into each project